### PR TITLE
2.3.3 minor

### LINKED ISSUE #351: _build in alt-ergo-lib-2.3.2 release
I am trying to build alt-ergo-lib.2.3.2 with dune.2.6.0 and Multicore OCaml 4.10.0 as part of https://github.com/ocaml-bench/sandmark. We use bwrap sandboxing to build packages in a different directory, but, I see that the 2.3.2 release already has a _build/ directory in the release package. Hence, I get the following error when trying to compile alt-ergo-lib.2.3.2:

```
# [ERROR] ../sandmark/_opam exists, but does not appear to be a valid opam root. Please remove it and use `opam init', or specify a different `--root' argument
# Exception: End_of_file.
```

Is it possible to provide a tarball without the _build directory, or is there a way to clean the _build directory before running `configure`?

### PR DESCRIPTION
**Pull request by OCamlPro-Coquera**
This Release is done to fix #351

### CODE REVIEW

File: sources/alt-ergo-lib.opam [hunk 1]
```diff
@@ -18,6 +18,9 @@ depends: [
   "seq"
   "stdlib-shims"
 ]
+conflicts: [
+  "alt-ergo-free"
+]
```
Comment (OriginLabs-Iguernlala) [inline sources/alt-ergo-lib.opam:23]: Should conflict with all (sub-) packages of Alt-Ergo-Free

File: sources/CHANGES [hunk 2]
```diff
@@ -1,6 +1,14 @@
 dev version
 ===========
 
+version 2.3.3, August 5, 2020
```
Comment (OriginLabs-Iguernlala) [inline sources/CHANGES:4]: Date is not accurate anymore :-)

**Review by OriginLabs-Iguernlala — COMMENTED** [review]
Don't forget to update the website as well

File: sources/alt-ergo-lib.opam [hunk 3]
```diff
@@ -18,6 +18,9 @@ depends: [
   "seq"
   "stdlib-shims"
 ]
+conflicts: [
+  "alt-ergo-free"
```
Comment (OCamlPro-Coquera) [inline sources/alt-ergo-lib.opam:22]:
```suggestion
  "alt-ergo-free"
  "alt-ergo-lib-free"
```

File: sources/alt-ergo-parsers.opam [hunk 4]
```diff
@@ -18,6 +18,9 @@ depends: [
   "menhir"
   "stdlib-shims"
 ]
+conflicts: [
+  "alt-ergo-free"
+]
```
Comment (OCamlPro-Coquera) [inline sources/alt-ergo-parsers.opam:23]:
```suggestion
```

File: sources/alt-ergo.opam [hunk 5]
```diff
@@ -15,6 +15,9 @@ depends: [
   "alt-ergo-lib" { = version }
   "alt-ergo-parsers" { = version }
 ]
+conflicts: [
+  "alt-ergo-free"
+]
```
Comment (OCamlPro-Coquera) [inline sources/alt-ergo.opam:20]:
```suggestion
```

File: sources/altgr-ergo.opam [hunk 6]
```diff
@@ -18,6 +18,9 @@ depends: [
   "alt-ergo-lib" { = version }
   "alt-ergo-parsers" { = version }
 ]
+conflicts: [
+  "alt-ergo-free"
+]
```
Comment (OCamlPro-Coquera) [inline sources/altgr-ergo.opam:23]:
```suggestion
```

File: sources/CHANGES (hunk 2 above)
Comment (OCamlPro-Coquera) [inline sources/CHANGES:4]:
```suggestion
version 2.3.3, August 19, 2020
```

**Review by OriginLabs-Iguernlala — APPROVED** [review]

**Pull request merged**